modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -3,56 +3,211 @@
 :toclevel: 2
 :openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
 :openid-connect-url: https://openid.net/connect/
+:openid-app-marketplace-url: https://marketplace.owncloud.com/apps/openidconnect
+:openid-app-github-url: https://github.com/owncloud/openidconnect
+:examplesdir: ../../../../examples/
 
 == Introduction
 
 "OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
 OpenID Connect allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. The specification suite is extensible, allowing participants to use optional features such as encryption of identity data, discovery of OpenID Providers, and session management, when it makes sense for them."
 -- {openid-connect-url}, "What is OpenID Connect?"
 
+//ownCloud-specific OIDC description and benefits
+== OIDC Benefits
+
+== OIDC App Marketplace Link
+
+// Not currently available
+* https://marketplace.owncloud.com/apps/openidconnect
+
 == Prerequisites
 
-A distributed Memcache setup is required to operate this app - such as Redis or Memcached properly.
-Please follow xref:configuration/server/caching_configuration.adoc[the documentation] on how to set up caching.
+The OpenID Connect app has several mandatory requirements, these are:
+
+* A minimum PHP version of 7.0, with the CURL and JSON extensions.
+* xref:configuration/server/caching_configuration.adoc[A distributed memory cache setup], such as _Redis_ or _Memcached_.
+This is because information about tokens need to be cached to prevent roundtrips to the identity provider. 
+Additionally, in clustered setups this information needs to be available for all nodes, which necessitates a distributed cache.
++
+NOTE: If a memory cache is not available, then you will be prevented from using the ownCloud UI, and see the following message: "_A real distributed mem cache setup is required_" in the ownCloud UI.
+
+== Recommended Options & Extensions
+
+The following PHP extensions are recommended by phpseclib, which the app is based on. 
+They are not mandatory.
+
+[cols="20%,80%"]
+|===
+|**ext-libsodium**
+|SSH2 can make use of some algorithms provided by the libsodium-php extension
```

Comment:
`libsodium` is as documented, already compiled in the required php version. Adding it additionally will give you error messages - see docs (my previous comments).
`php-ssh2` is currently _not_ part of our php installation routine, but this can be added easily in the prerequisites. Adding it additionally here may be confusing.
I recommend that you link to https://doc.owncloud.com/server/10.4/admin_manual/installation/server_prep_ubuntu_18.04.html